src/Compute/Compute/Generated/GalleryImage/GalleryImageCreateOrUpdateMethod.cs

```diff
@@ -185,7 +188,19 @@ public override void ExecuteCmdlet()
 
                     if (this.IsParameterBound(c => c.Feature))
                     {
-                        galleryImage.Features = this.Feature;
```

Comment:
changing from direct assign, to make it easier to add another feature when needed below